src/test/scala/org/ergoplatform/nodeView/WrappedUtxoState.scala

```diff
@@ -38,7 +35,9 @@ class WrappedUtxoState(override val version: VersionTag,
   override def applyModifier(mod: ErgoPersistentModifier): Try[WrappedUtxoState] = super.applyModifier(mod) match {
     case Success(us) =>
       mod match {
-        case ct: TCPMOD =>
+        case ct: TransactionsCarryingPersistentNodeViewModifier[ErgoTransaction@unchecked]
+          if ct.transactions.forall(_.isInstanceOf[ErgoTransaction]) =>
```

Comment:
Why to perform this check? You can not get block with transactions not being of ErgoTransaction type.